modules/imgcodecs/test/test_gif.cpp

```diff
@@ -246,13 +246,15 @@ TEST(Imgcodecs_Gif, read_gif_special){
     cv::Mat png_img1;
     ASSERT_NO_THROW(png_img1 = cv::imread(png_filename1,IMREAD_UNCHANGED));
     ASSERT_FALSE(png_img1.empty());
+    cv::cvtColor(png_img1, png_img1, cv::COLOR_BGRA2BGR);
```

Comment:
the color conversion is not needed, if you use imread without flags.

Reply:
'special1.png' and `special2.png` have alpha channel, but ‘special1.gif' and `special2.png` do not.
Having different arguments to imread() for each image type would have obscured the purpose of the test, 
so I modified it so that all imread()s use the same arguments `cv::IMREAD_COLOR`.